Source/Meadow.Foundation.Peripherals/Sensors.Distance.Hcsr04/Driver/Sensors.Distance.HCSR04/HCSR04.cs

```diff
@@ -51,16 +46,12 @@ public class Hcsr04 : IRangeFinder
 
         protected long tickStart;
 
-        
-
-        
-
         /// <summary>
         /// Create a new HCSR04 object with an IO Device
         /// </summary>
         /// <param name="triggerPin"></param>
         /// <param name="echoPin"></param>
-        public Hcsr04(IIODevice device, IPin triggerPin, IPin echoPin) :
+        public Hcsr04(IMeadowDevice device, IPin triggerPin, IPin echoPin) :
```

Comment:
I wonder if we should consider creating an `IGpioDevice` which is:

```
public interface IGpioDevice : IDigitalInputDevice, IDigitalOutputDevice {..}
```

I think the MCP23x08 would use it too.

Reply:
Fixed.